projects/plutus/inc/current_sense.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+// Current sense module for the LTC2484 ADC
```

Comment:
Please add any modules that are required to be initialized and improve your documentation

(ex. usage, side effects, etc.)